UE4SS/src/LuaType/LuaUEnum.cpp

```diff
@@ -110,16 +110,199 @@ No overload found for function 'UEnum.ForEachName'.
+
+            lua_object.get_remote_cpp_object()->InsertIntoNames(pair, param_index, true);
+            return 1;
+        });
+
+        table.add_pair("EditNameAt", [](const LuaMadeSimple::Lua& lua) -> int {
+            // Function: EditNameAt
+            // Param #1: Enum key index
+            // Param #2: New name for index
+            std::string error_overload_not_found{R"(
+No overload found for function 'UEnum.EditNameAt'.
+Overloads:
+#1: EditNameAt(integer Index, string NewName))"};
+
+            int32_t stack_size = lua.get_stack_size();
+
+            if (stack_size <= 0)
+            {
+                lua.throw_error("Function 'UEnum.EditValueAt' cannot be called with 0 parameters.");
+            }
+
+            int param_index = 0;
+            std::string param_new_name = L"";
+
+            // P1 (Index), integer
+            if (lua.is_integer())
+            {
+                param_index = lua.get_integer();
+            }
+            else
+            {
+                lua.throw_error("ERROR: Could not load parameter for \"Index\"");
+            }
+
+            // P2 (NewName), string
+            if (lua.is_integer())
+            {
+                param_new_name = to_wstring(lua.get_string());
+            }
+            else
+            {
+                lua.throw_error("ERROR: Could not load parameter for \"NewName\"");
+            }
+
+            auto& lua_object = lua.get_userdata<UEnum>();
+            Unreal::FName new_key = Unreal::FName(param_new_name, Unreal::FNAME_Add);
+            lua_object.get_remote_cpp_object()->EditNameAt(param_index, new_key);
+            return 1;
+        });
+
+        table.add_pair("EditValueAt", [](const LuaMadeSimple::Lua& lua) -> int {
+            // Function: EditValueAt
+            // Param #1: Enum key Index
+            // Param #2: New value for Index
+            std::string error_overload_not_found{R"(
+No overload found for function 'UEnum.EditValueAt'.
+Overloads:
+#1: EditValueAt(integer Index, integer NewValue))"};
+
+            int32_t stack_size = lua.get_stack_size();
+
+            if (stack_size <= 0)
+            {
+                lua.throw_error("Function 'UEnum.EditValueAt' cannot be called with 0 parameters.");
+            }
+
+            int param_index = 0;
+            int param_new_value = 0;
+
+            // P1 (Index), integer
+            if (lua.is_integer())
+            {
+                param_index = lua.get_integer();
+            }
+            else
+            {
+                lua.throw_error("ERROR: Could not load parameter for \"Index\"");
+            }
+
+            // P2 (NewValue), integer
+            if (lua.is_integer())
+            {
+                param_new_value = lua.get_integer();
+            }
+            else
+            {
+                lua.throw_error("ERROR: Could not load parameter for \"NewValue\"");
+            }
+
+            auto& lua_object = lua.get_userdata<UEnum>();
```

Comment:
This probably causes a crash, and if not, at least the parameter values won't be correct.
This should be the first thing retrieved from Lua, move this to the top of the function.
We always retrieve parameters (including the implicit this/context/self param) in the correct order.